server/graphql/v1/queries.js

```diff
@@ -349,6 +351,7 @@ const queries = {
       // Generate invoice
       const invoice = {
         title: get(transaction.host, 'settings.invoiceTitle'),
+        extraInfo: get(transaction.host, 'settings.invoice.extraInvoice'),
```

Comment:
```suggestion
        extraInfo: get(transaction.host, 'settings.invoice.extraInfo'),
```

Reply:
This was already reported and marked as resolved in https://github.com/opencollective/opencollective-api/pull/3431#discussion_r385024078

Reply:
im sorry that must have been a typo i have applied the suggested change